src/engine/strat_engine/backstore/setup.rs

```diff
@@ -282,18 +282,18 @@ pub fn get_blockdevs(
 
         if !duplicate_uuids.is_empty() {
             let err_msg = format!(
-                "The following list of Stratis UUIDs were each claimed by more than one Stratis device: {:?}",
-                duplicate_uuids
+                "The following list of Stratis UUIDs were each claimed by more than one Stratis device: {}",
```

Comment:
```format!("{}", p)``` is equivalent to ```p.to_string()```.

A further consideration is that, in most of its own code, stratisd uses the simple (unhyphenated) UUID format. This includes in implementation of Debug and also log messages. The only place where this appears to be not true is one place in the sim engine. Better to be consistent with the rest and use:

```p.simple().to_string()``` in place of ```format!("{}", p)```.

Reply:
Thanks, updated as expected.